README.md

```diff
@@ -108,34 +108,16 @@ An easier approach is likely to install Docker (https://www.docker.com/) and use
 
 Epic for making setting up a dev enviorment easier: https://pihemr.atlassian.net/browse/UHM-4245
 
-### Step 1: Clone the "mirebalais-puppet" project
-The various configuration files that determine what applications and options are turned on on different servers are
-found here and later on in process you will need to tell OpenMRS where to find them.
 
-```
-$ git clone https://github.com/PIH/mirebalais-puppet.git
-```
-
-### Step 2: (If you are directly installing MySQL on your machine) Ensure that MySQL has a password set up for the root user.
+### Step 1: (If you are directly installing MySQL on your machine) Ensure that MySQL has a password set up for the root user.
 - If you are able to run ```$ mysql -u root``` and access the MySQL Monitor without receiving an access denied error,
 it means that there is no root password set and you have to set it following the instructions here: https://dev.mysql.com/doc/refman/5.6/en/resetting-permissions.html
 - Once the root password has been set, you should be able to access the MySQL Monitor by running:  
   ```$ mysql -u root -p``` followed by entering the password when prompted.
 
-### Step 3: Add repositories to Maven settings file
-Find your maven `settings.xml` file. On Linux it should be at `~/.m2/settings.xml`. Add, in the `<repositories>` section,
-```xml
-<repository>  <!-- Mekom repository, for Initializer -->
-    <id>mks-nexus-public</id>
-    <url>https://nexus.mekomsolutions.net/repository/maven-public/</url>
-    <snapshots>
-        <updatePolicy>always</updatePolicy>
-    </snapshots>
-</repository>
-```
```

Comment:
This seems to be working for me without adding this, it looks like the mirroring Cintia has set up has resolved this issue.